test/sass/engine_test.rb

```diff
@@ -894,7 +900,7 @@ def test_properties_with_space_after_colon
 CSS
 foo
   bar : baz
-  bizz	: bap
+  bizz : bap
```

Comment:
I don't know why diff marks this as a change, I haven't touched this line :/

Reply:
the original file has a tabulation after `bizz`, the edited file has a space. Maybe the text editor converted that tabulation automatically?